File: cpp/tabliczka.cpp
/*
 * tabliczkacpp.cxx
 * 
 * Copyright 2018  <>
 */


#include <iostream>

using namespace std;

void tabliczka(int x, int y) {
    for (int i = 1; i <= x; i ++) {
        for (int j = 1; j <= y; i++) {
            cout << 1 * j << ' ';
        }
    }
}

int main(int argc, char **argv)
{   
    int m;
	m = 0; //inicjacja
    
    
    int n = 0;
    
    cout << "Podaj 1. liczbe: ";
    cin >> m;
    cout << "Podaj 2.liczbe: ";
    cin >> n;
    tabliczka(m, n);
    
	return 0;
}

File: python/tabliczka.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  potega.py
#  

def tabliczka(m, n):
    for i in range(1, m+1):
        for j in range(l, n + 1):
            print("{:>4}" .format(i * j),end='')
        print()
def main(args):
    m=int(input("Podaj 1.liczbe:"))
    n=int(input("Podaj 2.liczbe:"))
    tabliczka(m,n)
    
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
